content-src/components/DiscoveryStreamComponents/Hero/Hero.jsx

```diff
@@ -1,15 +1,51 @@
 import {connect} from "react-redux";
+import {DSCard} from "../DSCard/DSCard.jsx";
```

Comment:
It looks like most components only use relative links if it's `./` and it also looks like the .jsx is always omitted.

I don't personally have an opinion, just noticing this.